### PR TITLE
Contributing guidelines

### PR DESCRIPTION
**Pull request by JoshuaLight**
Note: this file was written by hand. It's pretty simple.

Addresses #51.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Creating an issue
+Use one of our issue templates to either submit a bug or request a feature.
+
+## Submitting a patch
+To submit a _minor_ change (bugfix), just create a pull request with a clear and concise description of what you're trying to address.
+
+To submit a _major_ change (feature, refactoring), link it with an open issue; consider creating one if needed.
+
+Please, follow our conventions through the path.
```
Comment (konstantin-ogulchansky) [inline CONTRIBUTING.md:14]: I think this sentence can be removed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Conventions
+
+We appreciate clean and elegant code, and we love to follow some conventions in our journey.
+
+### Coding
+- Use docstrings for each new class or function.
+- Use [Google Style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstrings.
+- Avoid creating utils or helper modules, classes or functions.
+- Avoid using `get` in method names. For example, `get_item` should be `item`.
+
```
Comment (JoshuaLight) [inline CONTRIBUTING.md:25]: Redundant blank line.

File: CONTRIBUTING.md (hunk 1 above)
Comment (JoshuaLight) [reply CONTRIBUTING.md]: Removed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (JoshuaLight) [reply CONTRIBUTING.md:21]: Fixed.

**Pull request merged**